monitor/api/alerts.py

```diff
@@ -59,23 +59,22 @@ def _query_alert_type(self, run_operation_args: dict, alert_factory_func: Callab
             logger.error('Failed to parse some alerts.')
             self.success = False
         return AlertsQueryResult(alerts, malformed_alerts)
-    
+
     @classmethod
     def _normalize_alert(cls, alert: dict) -> dict:
         try:
             normalized_alert = copy.deepcopy(alert)
-            test_meta = try_load_json(normalized_alert.get('test_meta'))
-            test_meta = test_meta if test_meta else {}
-            model_meta = try_load_json(normalized_alert.get('model_meta'))
-            model_meta = model_meta if model_meta else {}
+            test_meta = try_load_json(normalized_alert.get('test_meta')) or {}
+            model_meta = try_load_json(normalized_alert.get('model_meta')) or {}
```

Comment:
cosmetics - maybe just call it meta and add a comment that it could be either a model meta or a snapshot meta

Reply:
I wanted to make a clear distinction between the model's meta and the test's.

Reply:
ok by me :)